Spanner/tests/System/OperationsTest.php

```diff
@@ -279,8 +243,12 @@ public function testReadWithRestrictiveDatabaseRole()
             $error = $e;
         }
 
-        $this->assertInstanceOf(ServiceException::class, $error);
-        $this->assertEquals($error->getServiceException()->getStatus(), 'PERMISSION_DENIED');
+        if ($expected === null) {
+            $this->assertEquals(self::$id1, $row['id']);
+        } else {
+            $this->assertInstanceOf(ServiceException::class, $error);
```

Comment:
Since the exception is caught as a `ServiceException`, this is not helpful, maybe removed.

Reply:
You can alternatively assert not null, but since you are fetching its status below, it maybe overkill too.

Reply:
Fix this here and at other places in this PR, I pointed at some of them but not all.

Reply:
sounds good, removed.